react/graphql/getB2BCustomFields.graphql

```diff
@@ -0,0 +1,12 @@
+query GetB2BCustomFields {
```

Comment:
Vtex is going complain if you don't have a provider specified on this query.

Reply:
✅